docs/DOCUMENTATION_GUIDELINE.md

```diff
@@ -123,9 +123,9 @@ explicitly depend on them, you have likely not designed it properly.
 
 #### Proc-Macros
 
-Note that there are special considerations when documenting proc macros. Doc links will appear to function _within_ your
+Note that there are special considerations when documenting proc macros. Doc links will appear to function *within* your
 proc macro crate, but often will no longer function when these proc macros are re-exported elsewhere in your project.
-The exception is doc links to _other proc macros_ which will function just fine if they are also being re-exported. It
+The exception is doc links to *other proc macros* which will function just fine if they are also being re-exported. It
```

Comment:
Just wondering why the preference of * over _

Reply:
Not sure about a particular reason but it's defined in: mhttps://github.com/paritytech/polkadot-sdk/blob/master/.github/.markdownlint.yaml